### PR TITLE
Write capture skill

### PR DESCRIPTION
**Pull request by Yudai-8-om**
## Description
Capture class is rewritten.
tick returns the list of actions the robot can take.
is_done function is copied from the action.

### CODE REVIEW

File: rj_gameplay/rj_gameplay/skill/capture.py [hunk 1]
```diff
@@ -30,6 +30,10 @@ def __init__(self):
         self.root = self.capture_behavior
         self.root.setup_with_descendants()
 
-    def tick(self, world_state: rc.WorldState, robot: rc.Robot) -> None:
-        self.root.tick_once(robot)
-        # TODO: change so this properly returns the actions intent messages
+    def tick(self, robot: rc.Robot, world_state: rc.WorldState) -> None:
+    	self.robot = robot
+        self.root.tick_once(self.robot, world_state)
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/capture.py]: Almost right, this should return a list of actions (despite the return type being None). See `skill/move.py` in gameplay_node_update: https://github.com/RoboJackets/robocup-software/blob/gameplay_node_update/rj_gameplay/rj_gameplay/skill/move.py

File: rj_gameplay/rj_gameplay/skill/capture.py [hunk 2]
```diff
@@ -30,6 +30,10 @@ def __init__(self):
...
+
+        
+    def is_done(self, world_state) -> bool:
+    	self.capture.is_done(world_state)
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/skill/capture.py:39]: This should return a boolean, as per the method header. 

Again see skill/move.py in gameplay_node_update: https://github.com/RoboJackets/robocup-software/blob/gameplay_node_update/rj_gameplay/rj_gameplay/skill/move.py

**Review by kfu02 — CHANGES_REQUESTED** [review]
Good start, left some comments on skill/capture.py. Also, I think the fumble/pass classifier code is from a different PR, could you remove those files from this PR? (Removing those files from this branch won't remove them from your `fumble_classifier` branch.)

File: rj_gameplay/rj_gameplay/skill/capture.py [hunk 3]
```diff
@@ -30,6 +30,10 @@ def __init__(self):
         self.root = self.capture_behavior
         self.root.setup_with_descendants()
 
-    def tick(self, world_state: rc.WorldState, robot: rc.Robot) -> None:
-        self.root.tick_once(robot)
-        # TODO: change so this properly returns the actions intent messages
+    def tick(self, robot: rc.Robot, world_state: rc.WorldState) -> None:
+    	self.robot = robot
```
Comment (HussainGynai) [inline rj_gameplay/rj_gameplay/skill/capture.py:34]: You need to fix the indentation here.

**Review by HussainGynai — APPROVED** [review]
Looks good, just get rid of the extra files, and the one indentation issue.

**Pull request closed without merging**